src/app/contextBroker/contextBroker.cpp

```diff
@@ -602,15 +605,19 @@ PaArgument paArgs[] =
 
 
 
-#define API_V2                                                                                                   \
-  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      entryPointsTreat                          }, \
-  { "*",      EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      badVerbAllFour                            }, \
-  { "GET",    ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      getEntities                               }, \
-  { "*",      ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      badVerbGetOnly                            }, \
-  { "GET",    IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     getEntity                                 }, \
-  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     badVerbGetOnly                            }, \
-  { "GET",    IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, getEntityAttribute                        }, \
-  { "*",      IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, badVerbGetOnly                            }
+#define API_V2                                                                                     \
+  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,       entryPointsTreat           }, \
+  { "*",      EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,       badVerbAllFour             }, \
+                                                                                                   \
+  { "GET",    ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,       getEntities                }, \
+  { "POST",   ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,       postEntities               }, \
```

Comment:
I guess that this line addition is the only actual change, all the rest if style change. Is that correct?